src/features/deposit/components/Form/DepositFormNear/index.tsx

```diff
@@ -81,13 +81,14 @@ export const DepositFormNear = ({
 
   useEffect(() => {
     if (asset) {
-      const tokenAdapter: BaseTokenInfo = {
+      const tokenAdapter: SwappableToken = {
```

Comment:
`SwappableToken` belongs to swap widget, I don't recommend using it elsewhere

Reply:
No, we have a predefined list of tokens, which will be extended later on. From the perspective of the token list, this must remain the same; the only differences could be in the internal identifiers.

Reply:
Exactly, it was exclusively created for swap widget. If your type has the same value then it is just nice coincedence.